docs/core/entity.md

```diff
@@ -125,9 +125,13 @@ class MySwitch(SwitchEntity):
     ...
 ```
 
-This does exactly the same as the first example. Properties that can be set
-like this, start with `_attr_` followed by the property name. For example,
-the `device_class` property, has the `_attr_device_class` class attribute.
+This does exactly the same as the first example but relies on a default 
+implementation of the property in the base class. The name of the attribute
+starts with `_attr_` followed by the property name. For example, the default
+`device_class` property returns the `_attr_device_class` class attribute.
+
+Not all entity classes support the `_attr_` attributes, please refer
+to the documentation dor the respective entity class for details.
```

Comment:
```suggestion
to the documentation or the respective entity class for details.
```

Reply:
Something in general worth considering from PR reviews (although more programming related):
- Example of class variable and instance variable, where to use each and mention the risk of wrong usage.
- Example of setting attribute at init and accessing the property after 

```python
class SomeEntity():
    _attr_available: bool = True  # This will be common to all instances

    def __init__():
        self._attr_available = False  # This overrides the default
        self._attr_is_on = device.get_state() # Note that this will run only at init and probably should not be here
    
        # The following should be avoided:
        if some_complex_condition and some_other_condition and something_is_none_and_only_valid_after_update and device_available:
            
    def update()
        if self.available  # Read current state, no need to prefix with _attr_
            # Update the entity
            device_update()
            
        if error:
            self._attr_available = False  # Set property value
            return

        # We don't need to check if device available here
        self._attr_is_on = device.get_state()  # Update "is_on" property
```

Reply:
I think this is good, but it should perhaps be added as part of a scaffold instead.

Maybe extend the `integration` scaffold to create a `sensor.py` or something?

Reply:
I think it is better to have it in docs and not in the scaffold script, however it is just a comment and is not a requirement for this PR

Reply:
The example is added in 2ca3cc4